### ISSUE TITLE
Shadows seem to jitter

### ISSUE THREAD

**Issue by CurryKitten** [CurryKitten/CurryKitten-Sim#1]
When using the sim, the shadows can sometimes jitter quite noticeably.  This is potential an artifact of using a very small Near clipping plane... which has to be done in order for the quad to be visible from the on-board camera.   Will investigate

**Comment by howels** [comment]
I think there is also some Z-fighting between textures and shadows.... On the road texture sometimes the texture shows the ground underneath and sometimes the road, this appears similar to what is happening with the shadows.

**Comment by CurryKitten** [comment]
I have fixed this, it seems to be an odd "feature" of unity and the use of a very small clipping plane in a camera.  The clipping plane culls geometry that doesn't fall within it's range - so the far value effectively gives draw range, whilst the Near value defines where it draws from.  We needed to draw from as close from the camera as possible so we can see the details of the quad.  A side effect of a small Near clipping plane is that the shadows jitter. There doesn't seem to be any reason for this, and anyone at Unity going to fix it any time soon.  If we increase the Near value the shadows don't jitter, but we lose the front of the quad... so the solution:

2 cameras overlayed on each other.  I first defined my close in camera which would just be responsible for the very close in stuff - essentially the props

![Screen Shot 2019-12-10 at 22 27 08](https://user-images.githubusercontent.com/12443664/70574908-22730480-1b9d-11ea-9fe3-fa54150d67fc.jpg)

I then created a second camera which sits in the same position as the near camera and runs the same code, but the near clipping value has been tuned to make sure the shadows don't shake and as a result the props are all but missing

![Screen Shot 2019-12-10 at 22 27 31](https://user-images.githubusercontent.com/12443664/70574996-5c440b00-1b9d-11ea-995c-acf13d25ca38.jpg)

With both cameras running at once, they draw a complete view of the scene, but stop the shadow jitters.

I'll make this as closed, it'll go into the 0.4a release
